hub/powertoys/run.md

```diff
@@ -240,6 +240,42 @@ Examples:
 - `) calendar week::3/27/2022` to show the calendar week for a date value.
 - `) unix epoch::3/27/2022 10:30:45 AM` to convert the given time and date value into a Unix epoch timestamp.
 
+#### Custom formats
+The plugin includes a setting where custom formats can be defined. Custom formats are entered in a multiline text box which accepts one format per line.
+
+**Please use the following syntax:**
+
+- `<Format name>=<Format pattern>` for using the local time.
+- `<Format name>=UTC:<Format pattern>` for using the Universal Time Convention (UTC).
+
+> [!NOTE]
+> - `Format name`: Every charter except the equal sign is supported.
+> - `Format pattern`: You can escape the pattern and the backslash itself as text by using a backslash as prefix.
+
+**Examples:**
+
+- `MyFormat=dd-MMMM-yyyy`
+- `MySecondFormat=dddd (Da\y nu\mber: DOW)`
+- `MyUtcFormat=UTC:hh:mm:ss`
+
+
+**Supported format pattern:**
+
+| Format pattern | Description |
+| :--- | :--- |
+| Standard pattern like `hh:mm:ss`. |  Please see this [page](https://learn.microsoft.com/dotnet/standard/base-types/custom-date-and-time-format-strings) for more information.
```

Comment:
This will fix the build suggestion.

```suggestion
| Standard pattern like `hh:mm:ss`. |  Please see this [page](/dotnet/standard/base-types/custom-date-and-time-format-strings) for more information.
```